.gitignore

```diff
@@ -59,3 +59,6 @@ static_build
 static_final
 static
 npm-debug.log
+vendor-local/src/legal-docs
```

Comment:
We can't add the ignore to the repo since we do need to update it sometimes. If you want to add the ignore just to your computer [it looks like ](https://help.github.com/en/articles/ignoring-files#explicit-repository-excludes)you can create a file called `.git/info/exclude`.

Reply:
Fixed